extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -920,20 +947,27 @@ static public function onChangesListInsertArticleLink( $list, &$articleLink, &$s
 				$title = $wm->getMetaTitle();
 				$titleText = $wm->getArticleTitle()->getPrefixedText();
 				$pageText = $wm->getMainPageText();
-				$class = '';
 
-				$articleLink = ' <a href="' . $link . '" class="' . $class . '" >' . $title . '</a> ' . wfMessage( static::getMessagePrefix( $rc->getAttribute( 'rc_namespace' ) ) . '-new-message', [ $titleText, $pageText ] )->parse();
+				$attribs = [
+					'href' => $link,
+					'title' => $title
+				];
+
+				$articleLink = ' ' . Html::element( 'a', $attribs, $title ) . ' ' .
+					$list->msg( static::getMessagePrefix( $rc->getAttribute( 'rc_namespace' ) ) . '-new-message' )
+						->params( $titleText, $pageText )
+						->parse();
 
-				# Bolden pages watched by this user
-				# Check if the user is following the thread or the board
-				$user = $app->wg->User;
-				if ( $wm->isWatched( $user ) || $wm->isWallWatched( $user ) ) {
-					$articleLink = '<strong class="mw-watched">' . $articleLink . '</strong>';
+				// VOLDEV-3: Bolden pages watched by this user
+				$user = $list->getUser();
+				$specialPageTitle = $list->getTitle();
+				if ( self::shouldBoldenFollowedLink( $user, $specialPageTitle, $wm ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)